### PR TITLE
Stats page alignment

### PR DESCRIPTION
**Pull request by saml33**
– Aligned page structure
– Added charts for total deposit/borrow value
– Re-styled charts

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5,7 +5,6 @@
 "101@^1.0.0", "101@^1.2.0":
   version "1.6.3"
   resolved "https://registry.yarnpkg.com/101/-/101-1.6.3.tgz#9071196e60c47e4ce327075cf49c0ad79bd822fd"
-  integrity sha512-4dmQ45yY0Dx24Qxp+zAsNLlMF6tteCyfVzgbulvSyC7tCyd3V8sW76sS0tHq8NpcbXfWTKasfyfzU1Kd86oKzw==
```
Comment (mschneider) [inline yarn.lock]: not sure what happened here, these hashes should not disappear. @tjshipe did you add them with a special command?

**Comment by mschneider** [comment]
I really like the aggregate display as USD, but the way you calculate it, is currently wrong. You basically take BTC amounts for every hour and then value them, with the current price for BTC. You should value each hour by the price for the asset at that hour. @nicholasgodfreyclarke should be able to give you this data. He has been collecting it for the Deposit & Withdrawal account stats.

**Comment by saml33** [comment]
@mschneider – Have updated the prices. Not sure what's going on with yarn.lock

**Comment by mschneider** [comment]
image

when the mouse is not on the graph itself but on the highlighted area the total value shows NaN.
when the mouse is outside the graph the total value is blurred.

can we instead show the current value in each case? it's all i want to see when I load the page at the beginning of the day :)

**Comment by saml33** [comment]
@mschneider – There was an error with the prices api. Have updated to use the current price if there is no match from the prices api. Nick is making some updates that will make this an unlikely occurrence.

**Pull request merged**